### DISCUSSION TITLE
Alternative for select to set start in relative

### DISCUSSION THREAD

**Discussion by MBakerSE** [ekutner/home-connect-hass#395] [Ideas]
Hi,
First of all - great custom component! Now to my question:
Previous versions of this component used the "numbers" entity to set the start_in_relative (delayed start) e.g. for my dishwasher. This was great, because I could set the exact time to when I wanted it to start based on some calculations i made (energy price, solar production etc.)
But now the delayed start is a select entity, which only allows some set values for selection.

Would it be possible to add both? A selector might be easier for some, but the 30 min timeslots are not as precise as a numbers entity.

Or is there some workaround which I'm missing?

**Comment by ekutner** [comment]
It's true that it used to be a number entity but it has been a select for well over a year, I believe. The appliances themselves typically have a 1h resolution if you set the delay at the appliance so 30min seem like a reasonable compromise. 
If it's really important to you to be more precise than that you can use the **Start Program** service and pass the delay parameter with the value you want.
